lib/utils.js

```diff
@@ -268,3 +268,10 @@ module.exports.deleteDir = function (dir_path) {
     fs.rmdirSync(dir_path);
   }
 };
+
+module.exports.screenshotOutputFolder = function (fileName) {
+  if (!fileName.includes('/')) {
```

Comment:
This causes problem on Windows.
Why didn't you use `path.sep`?

Reply:
I missed it. will fix

Reply:
Great, @PeterNgTr !
When could we have a new build for this fix?
I've just upgraded CodeceptJS version today and going to downgrade it now 😢

Reply:
https://github.com/Codeception/CodeceptJS/pull/1395
I fixed it here, can you try it on windows. I don't have windows machine :)

new build push should depend on @DavertMik

Reply:
Glad to know version 2.0.1 has your fix.
Thanks 😄